### DISCUSSION TITLE
DatePicker: show two months

### DISCUSSION THREAD

**Discussion by Kettss1** [shadcn-ui/ui#3803] [Q&A]
Hey!
I need the Date Picker to show two months (it'll be the limit for selecting), but this will not be a RANGE selector. I just need to see two months at once. Is there a way to do it?
Thanks in advance!

**Comment by anishshobithps** [comment]
yes @Kettss1 , as per the [react-day-picker documentation](https://react-day-picker.js.org/api/interfaces/DayPickerBase#numberofmonths) you can allow to show many months you wanna display.

In the `calendar.tsx`, pass `numberOfMonths`, 
```ts
<DayPicker
  numberOfMonths={2}
  //...rest of the code
 ```
You can customize it and also try to pass it as props, so if some place if you required or needed only one month.

![image](https://github.com/shadcn-ui/ui/assets/38991749/488c9bbb-7dad-4cca-9f4a-054188b56e63)

I hope this solves your query ?
